wled00/cfg.cpp

```diff
@@ -868,6 +872,7 @@ void serializeConfig() {
   JsonObject hw_relay = hw.createNestedObject(F("relay"));
   hw_relay["pin"] = rlyPin;
   hw_relay["rev"] = !rlyMde;
+  hw_relay["odrain"] = rlyOpenDrain;
```

Comment:
use F()

Reply:
ok, should I change "pin" and "rev" also?

Reply:
No. Strings that are short and have many repetitions should not be enclosed in F() macro as they are optimised by linker.